runtime/gc_base/GCExtensions.cpp

```diff
@@ -256,8 +254,8 @@ MM_GCExtensions::computeDefaultMaxHeap(MM_EnvironmentBase *env)
 	}
 
 #if defined(OMR_ENV_DATA64)
-	if (J2SE_VERSION((J9JavaVM *)getOmrVM()->_language_vm) >= J2SE_V11) {
-		/* extend java default max memory to 25% of usable RAM */
+	if (!eanbleOriginalJDK8HeapSizeCompatibilityOption) {
+	/* extend java default max memory to 25% of usable RAM */
```

Comment:
The comment should be indented the same as the code in the block (as it was before).

Reply:
@LinHu2016 I launched Linux x86 extended test build. Would you please wait with this comment fix until it complete and results are reviewed?

Reply:
@dmitripivkine Sure

Reply:
Ok, test build failed because of Jenkins issues... Please submit the fix and we will start testing over